website/docs/r/virtual_network_gateway.html.markdown

```diff
@@ -221,23 +225,36 @@ The `vpn_client_configuration` block supports:
     The supported values are `SSTP`, `IkeV2` and `OpenVPN`.
     Values `SSTP` and `IkeV2` are incompatible with the use of 
     `aad_tenant`, `aad_audience` and `aad_issuer`.
+  
+---
 
 The `bgp_settings` block supports:
 
 * `asn` - (Optional) The Autonomous System Number (ASN) to use as part of the BGP.
 
-* `peering_address` - (Optional) The BGP peer IP address of the virtual network
-    gateway. This address is needed to configure the created gateway as a BGP Peer
-    on the on-premises VPN devices. The IP address must be part of the subnet of
-    the Virtual Network Gateway. Changing this forces a new resource to be created.
+* `peering_addresses` - (Optional) A list of `peering_addresses` as defined below. You can specify either one `peering_addresses` (normal use case) or two (when `active_active` is `true`).
 
 * `peer_weight` - (Optional) The weight added to routes which have been learned
     through BGP peering. Valid values can be between `0` and `100`.
+  
+---
 
 A `custom_route` block supports the following:
 
 * `address_prefixes` - (Optional) A list of address blocks reserved for this virtual network in CIDR notation. Changing this forces a new resource to be created.
 
+---
+
+A `peering_addresses` supports the following:
+
+* `ip_configuration_name` - (Required) The name of the IP configuration of this Virtual Network Gateway (as you defined in the`ip_configuration`).
```

Comment:
if this is defined in the `ip_configuration` for the gateway couldn't we just pull it from that property?

Reply:
In case there are multiple `peering_addresses`(s), in which case there are also multiple `ip_configuration`(s), you need to specify which `ip_configuration_name` this setting correspond to.

Reply:
if there is 1 and 1 we should be able to infer this and error out of we can't?